asm/include/overlay_41.inc

```diff
@@ -27,18 +27,18 @@
 .public GfGfxLoader_LoadCharDataFromOpenNarc
 .public GfGfxLoader_GXLoadPalFromOpenNarc
 .public GfGfxLoader_LoadFromOpenNarc
-.public sub_02007FD4
-.public sub_02008120
-.public sub_02008524
-.public sub_020085EC
-.public sub_02008780
-.public sub_020087A4
-.public sub_02008A78
-.public sub_020093FC
-.public sub_02009408
-.public sub_02009418
-.public sub_020094B0
-.public sub_02009D28
+.public PokepicManager_Create
+.public PokepicManager_DrawAll
+.public PokepicManager_Delete
+.public PokepicManager_CreatePokepic
+.public Pokepic_Delete
+.public Pokepic_SetAttr
+.public Pokepic_GetAttr
+.public PokepicManager_SetCharBaseAddrAndSize
+.public PokepicManager_SetPlttBaseAddrAndSize
+.public PokepicManager_HandleLoadImgAndOrPltt
+.public PokepicManager_SetNeedG3IdentityFlag
+.public UnscanPokepic
```

Comment:
Pokepic_Unscan

Reply:
this does not operate on the Pokepic class